frontend/src/routes/Credentials/CredentialsForm.tsx

```diff
@@ -250,6 +251,15 @@ export function CredentialsForm(props: {
     // Red Hat Cloud
     const [ocmAPIToken, setOcmAPIToken] = useState(providerConnection?.stringData?.ocmAPIToken ?? '')
 
+    function removeTrailingChar(url: string, char: string): string {
+        let newUrl = url
+        if (newUrl.charAt(newUrl.length - 1) === char) {
+            newUrl = newUrl.substr(0, newUrl.length - 1)
+            return removeTrailingChar(newUrl, char)
+        }
+        return newUrl
+    }
+
```

Comment:
When building the request url we assume the credential host does not contain a trailing '/', this function ensures that it is saved without one.

Reply:
We already depend on `lodash` so should we just use https://lodash.com/docs/4.17.15#trimEnd ?

Reply:
Also, if you parse the URL, note that it allows whitespace before and after, so it would be a good idea to trim whitespace as well. This is handy if someone does a copy-paste and accidentally includes some space.